Riot/AppDelegate.m

```diff
@@ -1622,7 +1825,15 @@ - (void)handleLimitedLocalNotifications:(MXSession*)mxSession events:(NSArray<NS
     for (NSString *eventId in events)
     {
         // Ignore event already notified to the user
-        if ([self displayedLocalNotificationForEvent:eventId andUser:userId type:nil])
+        if (@available(iOS 10, *))
+        {
+            if ([self displayedNotificationRequestForEvent:eventId andUser:userId type:nil])
```

Comment:
Like said above:
We are not obliged to check for displayed notifications here, as UNNotificationRequest have a unique identifier (evendId). The content for a UNNotificationRequest with the same identifier will be overwritten.

Reply:
https://github.com/vector-im/riot-ios/pull/2207/commits/04e9bf9a6b8e768f1a957ef7dff592572c98748c